Plugin/ProductList/Widget.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare(strict_types=1);
+
+namespace Bazaarvoice\Connector\Plugin\ProductList;
+
+use Bazaarvoice\Connector\Model\Source\ProductList;
+
+/**
+ * Class Widget
+ *
+ * @package Bazaarvoice\Connector\Plugin\ProductList
+ */
+class Widget extends Item
+{
+    /**
+     * @var string
+     */
+    protected $type = ProductList::WIDGET;
+
+    // @codingStandardsIgnoreStart
+
+    /**
+     * @param $subject
+     * @param $product
+     */
+    public function beforeGetProductPriceHtml(
```

Comment:
Use around plugin?

Reply:
I do not see a reason why not. This code was originally written by StoreFront. I have added a todo to test combining these into an around plugin.

Reply:
I do not think so, generateFeed does not throw an exception.